Wrapping/Generators/Python/itk/support/extras.py

```diff
@@ -385,6 +388,69 @@ def GetImageViewFromArray(arr, is_vector: bool = False, ttype=None):
 image_view_from_array = GetImageViewFromArray
 
 
+def array_from_vector_container(container, ttype=None):
+    """Get an Array with the content of the vector container"""
+    import itk
+
+    # Find container type
+    if ttype is not None:
+        if isinstance(ttype, (tuple, list)):
+            if len(ttype) != 1:
+                raise RuntimeError("Expected 1 component in ttype tuple.")
+            DataType = ttype[0]
+        else:
+            DataType = ttype
+    else:
+        DataType = itk.template(container)[1][1]
+    keys = [k for k in itk.PyVectorContainer.keys() if k[0] == DataType]
+    if len(keys) == 0:
+        raise RuntimeError("No suitable template parameter can be found.")
+    # Create numpy array of the type of the input container
+    return itk.PyVectorContainer[keys[0]].array_from_vector_container(container)
+
+
+def array_view_from_vector_container(container, ttype=None):
+    """Get an Array view with the content of the vector container"""
+    import itk
+
+    # Find container type
+    if ttype is not None:
+        if isinstance(ttype, (tuple, list)):
+            if len(ttype) != 1:
+                raise RuntimeError("Expected 1 component in ttype tuple.")
+            DataType = ttype[0]
+        else:
+            DataType = ttype
+    else:
+        DataType = itk.template(container)[1][1]
+    keys = [k for k in itk.PyVectorContainer.keys() if k[0] == DataType]
+    if len(keys) == 0:
+        raise RuntimeError("No suitable template parameter can be found.")
+    # Create numpy array of the type of the input container
+    return itk.PyVectorContainer[keys[0]].array_view_from_vector_container(container)
+
+
+def vector_container_from_array(arr, ttype=None):
+    """Get an Array view with the content of the vector container"""
```

Comment:
docstring is wrong due to copy-paste error

Reply:
Good catch! Fixed.